cfgov/jinja2/v1/_includes/organisms/post-preview.html

```diff
@@ -103,8 +103,8 @@
                     {{ date_desc }}
                     {% if 'EventPage' in post.specific_class.__name__ %}
                         {{ time.render(post.specific.start_dt, {'date':true}) }}
-		    {% elif 'EnforcementAction' in post.specific_class.__name__ and post.date_filed %}
-                        {{ time.render(post.date_filed, {'date':true}) }}
+		                {% elif 'EnforcementAction' in post.specific_class.__name__ and post.initial_filing_date %}
```

Comment:
Should this big indent be here?

Reply:
Yuck, there's tabs afoot. Fixed in 47fc5be